packages/vite/src/node/plugins/dynamicImportPolyfill.ts

```diff
@@ -3,7 +3,7 @@ import { Plugin } from '../plugin'
 import { isModernFlag } from './importAnalysisBuild'
 import path from 'path'
 
-export const polyfillId = 'vite/dynamic-import-polyfill'
+export const dynamicImportPolyfillId = 'vite/dynamic-import-polyfill'
```

Comment:
Is this rename necessary?

Reply:
Yes, this variable is imported together with the new polyfill so it is better they have distinct names